tests/FSharp.Compiler.ComponentTests/FSharpChecker/CommonWorkflows.fs

```diff
@@ -132,3 +139,34 @@ let ``Using getSource and notifications instead of filesystem`` () =
         checkFile middle expectSignatureChanged
         checkFile last expectSignatureChanged
     }
+
+[<Fact>]
+let GetAllUsesOfAllSymbols() =
+    let traceProvider =
+        Sdk.CreateTracerProviderBuilder()
+                .AddSource("fsc")
+                .SetResourceBuilder(ResourceBuilder.CreateDefault().AddService(serviceName="F#", serviceVersion = "1"))
+                .AddJaegerExporter()
+                .Build()
+
+    use _ = Activity.start "GetAllUsesOfAllSymbols" [  ]
+    
+    let result = 
+        async { 
+            let project = makeTestProject()
+            let checker = ProjectWorkflowBuilder(project, useGetSource=true, useChangeNotifications = true).Checker
+            do! saveProject project false checker 
+            let options = project.GetProjectOptions checker
+            // We should not have to do this:
+            //for p, f in project.GetAllFiles() do
+            //    let fn = p.ProjectDir ++ f.FileName
+            //    do! checker.NotifyFileChanged(fn, options) 
+            let! checkProjectResults = checker.ParseAndCheckProject(options) 
+            return checkProjectResults.GetAllUsesOfAllSymbols()
+        } |> Async.RunSynchronously
+
+
+    traceProvider.ForceFlush() |> ignore
```

Comment:
Why this?

Reply:
I copy pasted the code from ProjectGeneration. Don't remember why I put it there... either I wasn't always getting the traces or I copy pasted it from somewhere else even then 😅